coalib/parsing/LineParser.py

```diff
@@ -148,12 +157,12 @@ def __get_section_name(self, line):
 
         return ''
 
-    def __extract_keys_and_value(self, line):
+    def __extract_keys_and_value(self, line, delimiters):
```

Comment:
add a doc comment here please, this method is getting more complicated, I'm not so enthusiastic about line 165 esp